server/src/shared/infra/db/entities/club.entity.ts

```diff
@@ -11,6 +13,33 @@ export class ClubEntity extends BaseEntity {
   @Property()
   description!: string
 
+  @Property()
+  size!: number
+
+  @Property({ nullable: true })
+  backgroundImageURL!: string | null
+
+  @Property({ nullable: true })
+  iconURL!: string | null
```

Comment:
What's the difference between decorating the property with `@Property({ nullable: true })` and then marking the type `iconURL!: string | null`?

Reply:
Removed "| null" in latest, it was to fix a type error that I later changed.